core/server/index.js

```diff
@@ -53,6 +54,16 @@ function init() {
         return settings.init();
     }).then(function () {
         debug('Update settings cache done');
+        // Initialize full internationalization for core
+        // if back-end translations are enabled
+        // (settings for language and theme available here;
+        // full internationalization for theme is done
+        // shortly after, when activating the theme)
+        if (i18n.locale() !== 'en' && i18n.backendTranslations()) {
```

Comment:
Initialising the i18n module twice is super weird and especially because we don't support backend translations.
I would highly suggest to remove the backend translation logic for now and we can add them in a separate PR later with a better integration.

Reply:
Done.